### DISCUSSION TITLE
General question on SourceMods

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#6148] [E3SM model help]
Hello!

A follow up on source code modifications in E3SM - I've noticed that if I copy new scripts into the SourceMods directory, they don't get built into the model. I have to directly replace the files in ${CASE_DIR}/build/cmake_build/ or in the SRCROOT. 

Am I missing something or are there other more efficient work arounds for source mods?

Thanks! 
Jacob

**Comment by jonbob** [comment]
Hi Jacob,

The typical SourceMods usage does not work for MPAS components in E3SM, but does for others. Sorry for the inconvenience, but your workflow changing files in SRCROOT is correct.

Jon
